.changes/next-release/Bug Fix-7118f9a6-0252-4f60-b61d-95251221c864.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "ECS: prevent command executing before node has loaded"
```

This changelog doesn't really mean anything without context. (btw, PR numbers can be included in the changelog message).

Can the wording be changed to be meaningful on its own, e.g.:

```suggestion
	"description": "ECS: avoid null reference error when loading nodes #2441"
```